src/vstest.console/CommandLine/InferHelper.cs

```diff
@@ -44,8 +44,8 @@ public Architecture AutoDetectArchitecture(List<string> sources, IDictionary<str
                         }
                         else
                         {
-                            //TODO what to do for js, appx and others? Using default for now.
-                            arch = Constants.DefaultPlatform;
+                            //TODO what to do for js, appx and others? Using AnyCPU for now.
```

Comment:
We should remove `TODO` please.

Reply:
Should we make the chutzpah tests in our acceptance suite fail with issues like this? Seems like warnings are a P0 scenario blocker.

Reply:
Validate nodejs, python, chutzpah, uwp with this change.